### DISCUSSION TITLE
Identify commonly used phrases in documents

### DISCUSSION THREAD

**Discussion by JesseWeinstein** [tosdr/edit.tosdr.org#989] [Ideas]
We should figure out (or get from elsewhere) an algorithm for identifying common phrases from a set of documents, then create a page listing them -- this would be a useful way to pick out particularly relevant passages to create Cases for.

**Comment by JCBerger** [comment]
This could also be used to provide people with examples. Or some sort of cheat-sheet to use rather than just using the drop-down menu. Let me know what you think, and I might be able to make it a side project.
